Jellyfin.Plugin.Bookshelf/Providers/OpfReader.cs

```diff
@@ -222,6 +219,20 @@ private string FindMainTitle()
             return titleSort;
         }
 
+        private void FindAuthors(MetadataResult<Book> book)
+        {
+            var resultElement = _document.SelectNodes("//dc:creator[@opf:role='aut']", _namespaceManager);
```

Comment:
Will this cause issues by using a more narrow filter?
I don't know anything about the OPF spec

Reply:
I updated it. OPF has optional roles for creators and contributors. I made it so the it gets all creators and if role is available it chooses the correct role like illustrator or translator. If no role available it defaults to author.